docs/shone/examples/simple_transmission.rst

```diff
@@ -0,0 +1,293 @@
+.. code-block:: python
+
+    Opacity.get_available_species()
+
+
+.. raw:: html
+
+    <br /><div><i>Table length=9</i>
+    <table id="table11585383056" class="table-striped table-bordered table-condensed">
+    <thead><tr><th>name</th><th>species</th><th>charge</th><th>line_list</th><th>path</th><th>index</th></tr></thead>
+    <tr><td>synthetic</td><td>synthetic</td><td>--</td><td>example</td><td>/Users/bmmorris/.shone/synthetic__example.nc</td><td>0</td></tr>
+    </table></div><br /><br />
+
+
+Let’s load the opacity file named "synthetic" that we created above:
+
+.. code-block:: python
+
+    # load the synthetic opacity file:
+    opacity = Opacity.load_species_from_name('synthetic')
+
+
+Interpolating opacities
+-----------------------
+
+Now we will create a just-in-time compiled opacity interpolator.
+`~shone.opacity.Opacity.get_interpolator` returns a *function* that takes three
+arguments – a wavelength array [µm], a temperature [K], and a pressure
+[bar] – and returns an array of opacities for each wavelength:
+
+.. code-block:: python
+
+    # get a jitted 3D interpolator over wavelength, temperature, pressure:
+    interp_opacity = opacity.get_interpolator()
+
+
+Let's compute the opacity at one temperature and pressure:
+
+.. code-block:: python
+
+    wavelength = np.linspace(1, 5, 500)  # [µm]
+    pressure = 1  # [bar]
+    temperature = 200  # [K]
+
+    example_opacity = interp_opacity(wavelength, temperature, pressure)
+    
+    plt.semilogy(wavelength, example_opacity, label=f"T={temperature} K")
+    plt.legend()
+    plt.gca().set(
+        xlabel='Wavelength [µm]',
+        ylabel='Opacity, $\kappa$ [cm$^2$ g$^{-1}$]'
+    )
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    from shone.opacity import Opacity, generate_synthetic_opacity
+
+    generate_synthetic_opacity()
+
+    # load the one opacity file:
+    opacity = Opacity.load_species_from_name('synthetic')
+
+    # get a jitted 3D interpolator over wavelength, temperature, pressure:
+    interp_opacity = opacity.get_interpolator()
+
+    wavelength = np.linspace(1, 5, 500)  # [µm]
+    pressure = 1  # [bar]
+    temperature = 200  # [K]
+
+    example_opacity = interp_opacity(wavelength, temperature, pressure)
+
+    plt.semilogy(wavelength, example_opacity, label=f"T={temperature} K")
+    plt.legend()
+    plt.gca().set(
+        xlabel='Wavelength [µm]',
+        ylabel='Opacity, $\kappa$ [cm$^2$ g$^{-1}$]'
+    )
+
+Now let’s specify an opacity for a gray cloud:
+
+.. code-block:: python
+
+    kappa_cloud = 5e-2  # [cm2/g]
+
+
+Suppose we want to compute transmission spectra for several atmospheric
+temperatures. The function ``interp_opacity`` does not take vector
+arguments for temperature or pressure, but we can easily vectorize it
+with `~jax.vmap` like this:
+
+.. code-block:: python
+
+    # interpolate for a range of wavelengths at one pressure and temperature:
+    temperature = np.array([200, 400, 600, 800])
+
+    example_opacity = vmap(
```

Comment:
This call to vmap doesn't work in the example if `from jax import numpy as jnp, vmap` has not been repeated.

Reply:
I added it!